src/components/footer.js

```diff
@@ -1,13 +1,20 @@
-import React from 'react';
-import logo from '../img/iRODS-logo.png';
-import { version } from '../../package.json';
+import React from "react";
+import logo from "../img/iRODS-logo.png";
+import { version } from "../../package.json";
```

Comment:
this is fine here - it's very clear that this is just formatting / quotes in an otherwise simple diff/commit.

generally, we want to keep formatting tweaks separate from code change commits so it's easier for forensics / historical inquiries later on.

no changes needed - unless you want to split these three lines out.   your choice.   if so, please attribute it to the housekeeping issue in this repo... https://github.com/irods/irods_client_zone_management_tool/issues/17

Reply:
Removed these changes to keep code formatting separate from actual code change commits.